scripts/projectView.js

```diff
@@ -1,87 +1,90 @@
-var projectView = {};
```

Comment:
My comments below are pretty nit-picky, which is a good thing: your code's pretty clean!